### ISSUE TITLE
Deleting NonRDFSource causes Deleting 'NonRDFSource' object has no attribute '_build_binary' Exception

### ISSUE THREAD

**Issue by helrond** [ghukill/pyfc4#95]
This seems to be caused by https://github.com/ghukill/pyfc4/blob/94645fa56ea963aab14daca07330a0ea742e4a4d/pyfc4/models.py#L1251, which is calling a non-existent `_build_binary` method. I'm not sure exactly what that method was intended to do, but I'm wondering if this can just be removed? May possibly be related to #62

**Comment by ghukill** [comment]
Yes, I think I see what happened here.  [This commit](https://github.com/ghukill/pyfc4/commit/2fe079f7d76acab0826f65cd686598adeec3802d) dropped the original [`_build_binary()` method](https://github.com/ghukill/pyfc4/blob/8a7029e4aa12b3faaa8523a139d213644bf80a82/pyfc4/models.py#L1615-L1634), in favor of a new `BinaryData` class that would handle the initialization of those attributes.

Unfortunately, it is a bit sloppy how the `delete()` method is using the initialization of attributes to effectively just clean them, and it slipped through tests (where binary objects were normally just dropped as the child of another, container object).

Removed referenes to `_build_binary` method, and instead, created `empty()` method for `BinaryData` that can be run.  It's probably overkill when an object has been deleted in FC, but the variable remains, to assume it will get much use, but just in case.

Bumped version to `v0.4`: https://pypi.org/project/pyfc4/0.4/  Will wait to close, but hopefully this fixes.

**Comment by helrond** [comment]
Yup, confirming this solves the issue!
